### ISSUE TITLE
Roam should support standard Markdown

### ISSUE THREAD

**Issue by jrk** [Roam-Research/issues#340]
What Roam calls "Markdown" is not actually Markdown in several key ways. It's great that it has extended features (math mode, among other things), as many Markdown implementations do, but it makes the completely bizarre decision to use `__double-underscores__` for italics instead of `_single underscores_` or `*asterisks*`. (Standard Markdown would treat double-underscores as bold just like double-asterisks; this is not terribly common in real-world use, but arguably should also be supported.)

- This makes totally breaks muscle memory for everyone used to actual Markdown
- It is a misleading misuse of the name Markdown.
- Most significantly: it means you can't copy/paste _actual Markdown_ into Roam—nor can you copy/paste Roam content into other Markdown tools—and get italics where you expect them.

The current choice seems bizarre (why do something arbitrarily different from the whole rest of the world?), but I realize it may be entrenched already with many users. But: **There should at least be an _option_ to use more standard Markdown syntax for one's graph.**

Bear is a good example of a tool which chose to customize its markup language to be a little simpler by default, but it just has an option to use standard Markdown instead. Users can just trivially toggle and choose for themselves.

#311 is part of this overall theme as well. It has already covered the ability to _create_ headings using standard Markdown markup (when typing normally, not just when pasting).

**Comment by stevelw** [comment]
This is so broken I think it's an (intentionally created) bug rather than a feature request.

**Comment by thomkeh** [comment]
> The current choice seems bizarre (why do something arbitrarily different from the whole rest of the world?)

I want to point out that Roam is certainly not the first to do this. Dynalist (which Roam is probably partly inspired by?) also does it this way [which they justify on their help page](https://help.dynalist.io/article/90-formatting-reference) with:

> Dynalist’s Markdown syntax is identical to Markdown except that `*italic*` and `_italic_` doesn’t work. Instead, `**bold**` and `__italic__` is used. The intention is to not surprise people who don’t use Markdown, as it’s common to have `*` or `_` surround text.

That said, I do agree with this feature request and would very much like to see pure markdown support.

**Comment by anEXPer** [comment]
Roam looks very interesting, but my investigation of it hit the wall of "oh, no option for actual markdown, too bad the search continues" within minutes.

I work with markdown in many other places. I need a consistent input experience, and I need the ability to copy and paste between systems. Github, Mkdocs, Discourse, Stack Overflow, etc, etc, all use markdown.

**Comment by nborson** [comment]
Agreed: please support [CommonMark](https://commonmark.org/). GitHub, Discourse, Reddit, and StackExchange do. It's crucial for me to be able to import into Roam from Pandoc markdown output. The italics issue is especially bothersome.
